src/components/feed-display/FeedItemDisplayList.vue

```diff
@@ -196,6 +196,10 @@ export default Vue.extend({
 		},
 		// Trigger the click event programmatically to benefit from the item handling inside the FeedItemRow component
 		clickItem(item: FeedItem, alignToTop = false) {
+			if (!item) {
```

Comment:
I did it because it was easier to check for it every time the method is used. And I got no warning before the fix (just the error) and no warning after the fix. ️🤷🏻